K/kdb_c_api/build_tarballs.jl

```diff
@@ -3,39 +3,39 @@
 using BinaryBuilder, Pkg
 
 name = "kdb_c_api"
-version = v"2022.7.19"
+version = v"2022.9.3"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/KxSystems/kdb.git", "01fb1243c6b32079938ef88a4434e46d2f3c00be")
+    GitSource("https://github.com/KxSystems/kdb.git", "f1b8e5e1dfaf86b1ccd3a2634d10e07718b71ece")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
-cd $WORKSPACE/srcdir
 install_license kdb/LICENSE
 mkdir ${libdir}
-if [[ ${target} == aarch64-linux-* ]]; then opath="kdb/l64arm/c.o"; fi
-if [[ ${target} == x86_64-linux-* ]]; then opath="kdb/l64/c.o"; fi
-if [[ ${target} == i686-linux-* ]]; then opath="kdb/l32/c.o"; fi
-if [[ ${target} == aarch64-apple-* ]]; then opath="kdb/m64/c.o"; fi
-if [[ ${target} == x86_64-apple-* ]]; then opath="kdb/m64/c.o"; fi
-if [[ ${target} == i686-apple-* ]]; then opath="kdb/m32/c.o"; fi
-if [[ ${target} == x86_64-w64-* ]]; then opath="kdb/w64/c.dll"; fi
-if [[ ${target} == i686-w64-* ]]; then opath="kdb/w32/c.dll"; fi
+if [[ ${target} == aarch64-linux-* ]];  then opath="kdb/l64arm/c.o"; fi
+if [[ ${target} == x86_64-linux-* ]];   then opath="kdb/l64/c.o"; fi
+if [[ ${target} == i686-linux-* ]];     then opath="kdb/l32/c.o"; fi
+if [[ ${target} == aarch64-apple-* ]];  then opath="kdb/m64arm/c.o"; fi
+if [[ ${target} == x86_64-apple-* ]];   then opath="kdb/m64/c.o"; CC="gcc"; fi
```

Comment:
Why using gcc?

Reply:
For whatever reason, `x86_64-apple-*` build would fail with clang default.

Reply:
Well, I'd like to know the reason

Reply:
I get the following error without `CC="gcc"`:
```Undefined symbols for architecture x86_64:
  "___emutls_get_address", referenced from:
      l004 in c.o
      l005 in c.o
      l006 in c.o
      _krr in c.o
      l038 in c.o
      l039 in c.o
      l035 in c.o
      ...
ld: symbol(s) not found for architecture x86_64
clang-13: error: linker command failed with exit code 1 (use -v to see invocation)
[✗]
```